attestation-service/attestation-service/src/bin/restful/mod.rs

```diff
@@ -0,0 +1,171 @@
+use std::sync::Arc;
+
+use actix_web::{body::BoxBody, web, HttpResponse, ResponseError};
+use anyhow::{bail, Context};
+use attestation_service::{policy_engine::SetPolicyInput, AttestationService, HashAlgorithm};
+use base64::{engine::general_purpose::URL_SAFE_NO_PAD, Engine};
+use kbs_types::Tee;
+use log::{debug, info};
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+use strum::AsRefStr;
+use thiserror::Error;
+use tokio::sync::RwLock;
+
+#[derive(Error, Debug, AsRefStr)]
+pub enum Error {
+    #[error("An internal error occured: {0}")]
+    InternalError(#[from] anyhow::Error),
+}
+
+impl ResponseError for Error {
+    fn error_response(&self) -> HttpResponse {
+        let body = format!("{self:#?}");
+
+        let mut res = match self {
+            Error::InternalError(_) => HttpResponse::InternalServerError(),
+            // _ => HttpResponse::NotImplemented(),
+        };
+
+        res.body(BoxBody::new(body))
+    }
+}
+
+type Result<T> = std::result::Result<T, Error>;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct AttestationRequest {
+    tee: String,
+    evidence: String,
+    runtime_data: Option<Data>,
+    init_data: Option<Data>,
+    runtime_data_hash_algorithm: Option<String>,
+    init_data_hash_algorithm: Option<String>,
+    policy_ids: Vec<String>,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
+enum Data {
+    Raw(String),
+    Structured(Value),
```

Comment:
I am ok with both. Why I use the name here is because we defined gRPC API in the last PR with name `raw_runtime_data` and `structured_runtime_data`.
https://github.com/confidential-containers/kbs/blob/main/attestation-service/protos/attestation.proto#L34-L43